client/app/account/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import React from 'react'
+import { FormContainer, Loading } from '@/_components/layout'
+import { Button, Heading } from '@/_components/ui'
+import { useGetMeQuery } from '@/_hooks/api'
+import { AccountInfo } from './component/AccountInfo'
+
+export default function Account() {
+  const { data: user, isLoading } = useGetMeQuery()
+
+  if (isLoading || !user) return <Loading />
+  /* TODO: error handling */
+
+  return (
+    <>
```

Comment:
You don't need this React Fragment!

Reply:
Fixed it! 
8b448ff822baa353a3e0b55f56b2897a2ba4f855